assets/components/openshift-dns/dns/configmap.yaml

```diff
@@ -16,7 +16,7 @@ data:
             fallthrough in-addr.arpa ip6.arpa
         }
         prometheus 127.0.0.1:9153
-        forward . /etc/resolv.conf {
+        forward . {{if eq .UpstreamResolver ""}}/etc/resolv.conf{{else}}{{.UpstreamResolver}}{{end}} {
```

Comment:
I see this changed compared to what was cherry-picked, is there a functional difference?

Reply:
Good catch! I missed backporting the other commit.
fixed.